communication/zoom-guidelines.md

```diff
@@ -217,7 +217,7 @@ Thanks for making Kubernetes meetings work great!
 [community meeting]: /events/community-meeting.md
 [SIG/WG meetings]: /sig-list.md
 [Office Hours]: /events/office-hours.md
-[Meet Our Contributors]: /mentoring/meet-our-contributors.md
+[Meet Our Contributors]: https://github.com/kubernetes/community/blob/master/mentoring/programs/meet-our-contributors.md
```

Comment:
```suggestion
[Meet Our Contributors]: /mentoring/programs/meet-our-contributors.md
```
Small nit to keep the relative link path format consistent :)
lgtm otherwise

Reply:
Yes, makes it easier to read. Made the change!